content/blog/2018/incremental-traffic-management/index.md

```diff
@@ -67,7 +67,6 @@ $ helm template install/kubernetes/helm/istio \
   --set sidecarInjectorWebhook.enabled=false \
```

Comment:
note the istio-remote chart has been deprecated and merged into the istio chart.  We should probably just remove istio-remote entirely from the repository as it is confusing and doesn't work as is.  I am struggling to find the exact flag - @ymesika knows - I believe he was doing this consolidation worrk.